deepmd/pd/model/descriptor/repflows.py

```
@@ -0,0 +1,546 @@
+        a_compress_rate: int = 0,
+        a_compress_e_rate: int = 1,
+        a_compress_use_split: bool = False,
+        n_multi_edge_message: int = 1,
+        axis_neuron: int = 4,
+        update_angle: bool = True,
+        activation_function: str = "silu",
+        update_style: str = "res_residual",
+        update_residual: float = 0.1,
+        update_residual_init: str = "const",
+        set_davg_zero: bool = True,
+        exclude_types: list[tuple[int, int]] = [],
```

Ruff (0.8.2)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)